azure-pipelines.yml

```diff
@@ -61,833 +52,190 @@ variables:
+    #  autoBaseline: true
+    sdl:
+      binskim:
+        enabled: true
+      policheck:
+        enabled: true
+      sbom:
+        enabled: false # VS SBOM is generated with other steps
+        justificationForDisabling: 'SBOM for F# is generated via build process.'
+      sourceAnalysisPool:
+        name: NetCore1ESPool-Svc-Internal
+        image: 1es-windows-2022-pt
+        os: windows
+    stages:
+    - stage: build
+      displayName: Build
+      jobs:
+      #-------------------------------------------------------------------------------------------------------------------#
+      #                                                  Signed build                                                     #
+      #-------------------------------------------------------------------------------------------------------------------#
+      # Localization: we only run it for specific release branches
+      - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/dev17.10') }}:
+        - template: /eng/common/templates-official/job/onelocbuild.yml@self
+          parameters:
+            MirrorRepo: fsharp
+            MirrorBranch: $(FSharpReleaseBranchName)
+            LclSource: lclFilesfromPackage
+            LclPackageId: 'LCL-JUNO-PROD-FSHARP'
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
         parameters:
-          MirrorRepo: fsharp
-          MirrorBranch: release/dev17.9
-          LclSource: lclFilesfromPackage
-          LclPackageId: 'LCL-JUNO-PROD-FSHARP'
-    - template: /eng/common/templates/jobs/jobs.yml
+          enableMicrobuild: true
+          enablePublishBuildArtifacts: true
+          enablePublishTestResults: false
+          enablePublishBuildAssets: true
+          enablePublishUsingPipelines: $(_PublishUsingPipelines)
+          enableSourceBuild: true
+          enableTelemetry: true
+          helixRepo: dotnet/fsharp
+          jobs:
+          - job: Full_Signed
+            pool:
+              name: NetCore1ESPool-Svc-Internal
+              image: windows.vs2022preview.amd64
+            timeoutInMinutes: 300
+            templateContext:
+              mb:
+                signing:
+                  enabled: true
+                  feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
+                  signType: $(_SignType)
+                  zipSources: false
+
+            variables:
+            - group: DotNet-Symbol-Server-Pats
+            - group: DotNet-DevDiv-Insertion-Workflow-Variables
+            - name: _SignType
+              value: Real
+            steps:
+            - checkout: self
+              clean: true
+            - template: /eng/restore-internal-tools.yml
+            - script: eng\CIBuild.cmd
+                      -configuration $(_BuildConfig)
+                      -prepareMachine
+                      -testAllButIntegration
+                      -officialSkipTests $(SkipTests)
```

Comment:
SkipTests doesn't seem to be initialized anywhere, probably can be removed.

Reply:
It can't it's passed in CI